build.xml

```diff
@@ -1700,7 +1700,7 @@ xmlns:cs="antlib:com.puppycrawl.tools.checkstyle.ant">
 
     <target name="owasp" depends="owasp-taskdef,ivy-retrieve" description="OWASP dependency check">
         <property name="owasp.out.dir" value="${test.java.build.dir}/owasp" />
-
+        <owasp:dependency-check-update xmlns:owasp="antlib:org.owasp.dependencycheck.anttasks" />
```

Comment:
this is to make CI download patterns
on CI we have a shared ant env and it seems not to download patterns automatically, the best is to force the download at every execution, it takes 10 minutes but it is not a problem

@phunt @anmolnar @hanm